lib/components/Charts/BarChart/index.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { BarChart } from "."
+
+const dataConfig = {
+  desktop: {
+    label: "Desktop",
+  },
+  mobile: {
+    label: "Mobile",
+  },
+}
+
+const Component = BarChart<typeof dataConfig>
+
+const meta: Meta<typeof Component> = {
+  component: Component,
+  title: "Charts/BarChart",
+  args: {},
+} satisfies Meta<typeof Component>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    dataConfig: {
+      desktop: {
+        label: "Desktop",
+        color: "hsl(var(--chart-1))",
+      },
+      mobile: {
+        label: "Mobile",
+        color: "hsl(var(--chart-2))",
+      },
+    },
+    xAxis: {
+      hide: false,
+    },
+    lines: true,
```

Comment:
Is that something we really need? From the screenshot, it looks a bit redundant.